capture/capture.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABCMeta, abstractmethod
+
+import capture
+
+class ImgCapture(metaclass=ABCMeta):
```

Comment:
Wow. If you say so. :-)
You're right about the intent, and your suggestion is indeed way simpler.

I just never used `ABCMeta` before and felt it was the occasion. :-P